destinationCreation.css

```
@@ -0,0 +1,5 @@
+#content {
```

```suggestion
/* TODO: Move common styles to a core css file. */
#content {
```

Because it will be good to use consistent page width, and probably other styles as time goes on, across all pages.